.opencode/commands/implement-oxlint-rules.md

```
@@ -0,0 +1,132 @@
+|---|---|---|
+| `eslint` | Top-level `rules` | `// ---- ESLint rules ----` |
+| `import` | Top-level `rules` | `// ---- Import rules ----` |
+| `node` | Top-level `rules` | `// ---- Node Rules ----` |
+| `oxc` | Top-level `rules` | `// ---- Oxc rules ----` |
+| `promise` | Top-level `rules` | `// ---- Promise Rules -----` |
+| `unicorn` | Top-level `rules` | `// ---- Unicorn Rules -----` |
+| `vue` | Top-level `rules` | `// ---- Vue Rules -----` |
+| `typescript` | First override (`**/*.ts`, `**/*.vue`) | `// ---- TypeScript rules ----` |
+| `vitest` | Spec override (`**/*.spec.ts`) | Within the vitest rules block |
+
+2. **For the `node` plugin (first time only):**
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)